src/OpenTracing/Standard/Format.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OpenTracing\Standard;
+
+class Format {
+
+	const SPLIT_TEXT = 'text';
```

Comment:
we may have non-split text in the future, so `'split_text'`, etc, likely makes more sense.